common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -12801,6 +12801,14 @@ public static TwoFactorAuthSecretEncoding fromString(String s) throws ServiceExc
     @ZAttr(id=511)
     public static final String A_zimbraPrefDisplayExternalImages = "zimbraPrefDisplayExternalImages";
 
+    /**
+     * Display received/sent time in mail list
+     *
+     * @since ZCS 8.8.8
```

Comment:
I have changed value of since attribute as we are going to ship this code in 8.8.8
@rupalid @k-kato @Prashantsurana